### ISSUE TITLE
Feedback or Time triggered workflow

### ISSUE THREAD

**Issue by alberk8** [elsa-workflows/elsa-core#4114]
Does elsa support the following.

1) Workflow Starts when it receive a message from a web api.  This then proceed to send a proceed to send a web api call to another app.  If the user does not respond in the positive in a predefined time then proceed to next workflow.

**Comment by sfmskywalker** [comment]
Yes, and depending on the details, there are different ways to go about this.
For example, you mention: "workflow starts when it receives a message rom a web api" - your web API controller could invoke a given workflow. This workflow can then fork into two branches: one that leads into ab HTTP Request activity to make another web API call, after which you wait for an event (using the Event activity) and another into a Delay activity that represents your timeout. Both branches should then merge into a Join activity (with wait mode set to "wait any" so that it will  remove the bookmarks from either the Delay or the Event activity) and continue.
